stewart/readme.txt

```diff
@@ -1,7 +1,7 @@
 === Stewart ===
 Contributors: automattic
 Requires at least: 5.8
-Tested up to: 5.9
+Tested up to: 5.8
```

Comment:
This can't be 5.9 if we're submitting the theme before 5.9 is actually released. So I'm putting 5.8 for now.